Classes/Emogrifier.php

```diff
@@ -148,6 +148,14 @@ class Emogrifier
     private $shouldKeepInvisibleNodes = true;
 
     /**
+     * Determines whether CSS styles that have an equivalent HTML attribute
+     * should be mapped and attached to those elements.
+     *
+     * @var bool
+     */
+    private $shouldMapCss2Html = false;
```

Comment:
Please don't use abbreviations: shouldMapCss2Html -> shouldMapCssToHtml

In addition, the mapping is to HTML _attributes_. So the naming should be mapCssToHtmlAttributes.